app/models/book.rb

```diff
@@ -3,6 +3,7 @@ class Book < ApplicationRecord
   has_many :users, through: :copies
```

Comment:
we also might want to add **has_many :copies** in the next PR

Reply:
It's already there, just one line above (line 2) :) thanks for merging!